tests/src/utils.rs

```diff
@@ -51,7 +53,6 @@ pub(crate) fn run_guest<T, OV>(
     );
 }
 
-// TODO: add test
```

Comment:
Missed removed comment in https://github.com/eth-act/zkevm-benchmark-workload/pull/156